client/state/data-layer/wpcom/sites/jitm/index.js

```diff
@@ -66,69 +64,32 @@ export const doFetchJITM = ( action ) =>
 	http(
 		{
 			method: 'GET',
-			path: `/jetpack-blogs/${ action.siteId }/rest-api/`,
+			apiNamespace: 'wpcom/v3',
+			path: `/sites/${ action.siteId }/jitm`,
```

Comment:
Here you'll need to add `isLocalApiCall` to prevent `jetpack_site_xhr_wrapper` from modifying the request prefix. And then in `data-layer/wpcom-http`, in the `queueRequest` function, catch `options.isLocalApiCall` and pass it to the `fetcher`. Very similar to the existing `responseType` option.

Reply:
I went down the indirection rabbit hole of http handlers and managed to work out where to pass the param.